aligner/prondict.py

```diff
@@ -42,25 +42,31 @@ class PronDict(object):
     @staticmethod
     def pronify(source):
         for (i, line) in enumerate(source, 1):
-            if line.startswith(";"):
+            line = line.strip()
+            if not line or line.startswith(";") or line.startswith("#"):
```

Comment:
If we want to extend the notion of what valid comments are, okay, but that seems totally separate from what we're doing here; could you revert and propose that elsewhere?

Semicolon comments in pronunciation dictionaries are a reasonably old convention from when all speech tech was in Scheme...the venerable CMUdict has comments in this form. I'm not 100% sure we still need support for them and maybe we should also remove that in later work.

Reply:
The `#` comment marker was part of the original proposal :-)

I can split this off. Given that `HDMan` doesn't appear to support `;` comments, shall I also remove that in the new PR?